spec/dry_run_spec.rb

```diff
@@ -1,10 +1,6 @@
 require 'spec_helper'
 
 describe Slackistrano do
-  before(:each) do
-    Rake::Task['load:defaults'].execute
```

Comment:
Thank you for pointed them out!

Hmm, I ran the spec following commands, and it was errors.

But .... now, I look it in more specifically, think this commit is unnecessary. :scream_cat:

```
$ bundle exec rspec spec/dry_run_spec.rb 
Run options: include {:focus=>true}

All examples were filtered out; ignoring {:focus=>true}

Randomized with seed 42008

Slackistrano
** Execute load:defaults
** Execute slack:deploy:updating
WARN [slackistrano] Error notifying Slack!
WARN [slackistrano]   Error: #<SocketError: Failed to open TCP connection to .slack.com:443 (getaddrinfo: nodename nor servname provided, or not known)>
  does not post to slack on slack:deploy:updating (FAILED - 1)

Failures:

  1) Slackistrano does not post to slack on slack:deploy:updating
     Failure/Error: if response.code !~ /^2/
     
     NoMethodError:
       undefined method `code' for nil:NilClass
     # ./lib/slackistrano/capistrano.rb:92:in `block in post'
     # ./lib/slackistrano/capistrano.rb:67:in `each'
     # ./lib/slackistrano/capistrano.rb:67:in `post'
     # ./lib/slackistrano/capistrano.rb:29:in `block in run'
     # ./.bundle/ruby/2.3.0/gems/sshkit-1.10.0/lib/sshkit/backends/abstract.rb:29:in `instance_exec'
     # ./.bundle/ruby/2.3.0/gems/sshkit-1.10.0/lib/sshkit/backends/abstract.rb:29:in `run'
     # ./.bundle/ruby/2.3.0/gems/capistrano-3.5.0/lib/capistrano/dsl.rb:58:in `run_locally'
     # ./lib/slackistrano/capistrano.rb:29:in `run'
     # ./lib/slackistrano/tasks/slack.rake:6:in `block (3 levels) in <top (required)>'
     # ./.bundle/ruby/2.3.0/gems/rake-11.1.2/lib/rake/task.rb:248:in `block in execute'
     # ./.bundle/ruby/2.3.0/gems/rake-11.1.2/lib/rake/task.rb:243:in `each'
     # ./.bundle/ruby/2.3.0/gems/rake-11.1.2/lib/rake/task.rb:243:in `execute'
     # ./spec/dry_run_spec.rb:12:in `block (3 levels) in <top (required)>'

Finished in 0.26051 seconds (files took 3.17 seconds to load)
1 example, 1 failure

Failed examples:

rspec './spec/dry_run_spec.rb[1:1]' # Slackistrano does not post to slack on slack:deploy:updating

Randomized with seed 42008
```